designs/index.md

```diff
@@ -14,27 +14,37 @@ changes as designs are implemented.
 Always go to the [documentation]({{ site.docs_site_url }}) for descriptions of
 current Bazel functionality.
 
-## Table of Contents
-{:.no_toc}
-
-* ToC
-{:toc}
-
-
 ## When do I need a design document and review?
 
 If you're planning to add, change, or remove a user-facing feature, or make a
-significant architectural change to Bazel, you **must** write a design document
-and have it reviewed before you can submit the change. See [What is a
-significant change?](#what-is-a-significant-change) for details.
+*significant architectural change* to Bazel, you **must** write a design document
+and have it reviewed before you can submit the change.
 
 Implementation can begin before the proposal is accepted, for example as a
 proof-of-concept or an experimentation. However, you cannot submit the change
 before the review is complete.
 
+## What is a significant change?
 
-## Workflow
+There are no hard and fast criteria, but here are some examples:
```

Comment:
Hard and fast is colloquial. What about 
"While there is no codified criteria for significant changes, here are some examples:"

Reply:
Alternatively, we can just update this section to be ##Significant change criteria 
Thoughts?

Reply:
Removed the header altogether. Is this better?